recipe_scrapers/lekkerensimpel.py

```diff
@@ -41,7 +73,11 @@ def cuisine(self):
         return self.schema.cuisine()
 
     def description(self):
-        return self.schema.description()
+        try:
+            return self.schema.description()
+        except SchemaOrgException:
+            description = self.soup.find("meta", {"name": "description"})
+            return description["content"] if description else None
```

Comment:
```suggestion
            pass

        description = self.soup.find("meta", {"name": "description"})
        return description["content"] if description else None
```

Rationale: attempting to separate exception-handling logic for one content retrieval approach from an alternate content retrieval approach.